main/updater.js

```diff
@@ -86,6 +94,7 @@ function onUpdaterError (err) {
  * @param {import('electron-updater').UpdateInfo} info
  */
 function onUpdateAvailable ({ version /*, releaseNotes */ }) {
+  ipcMain.emit(ipcMainEvents.UPDATE_AVAILABLE)
```

Comment:
Based on the code below, I think we should show emit UPDATE_AVAILABLE event only after we have successfully downloaded the new version.

Since the app will display a banner at the top, I think we should also stop calling `showDialogSync`.

Also: the dialog has two buttons - Close and Show Release Notes. For the in-app notification, we arse showing Restart instead of Close, which is fine. 

Can we please keep the UX for showing release notes? For example, the in-app notification can show two buttons - "Restart" and "Show Release Notes".

Reply:
I'd say pick the option that's easiest to implement. Opening the release notes in the default web browser is good enough, IMO.

Reply:
I added a link for release notes on [900758d](https://github.com/filecoin-station/filecoin-station/pull/199/commits/900758d4d5d42c44576bb376a9981cd4251f5aeb)

Since we're on an electron window, I implemented an RPC so electron can trigger the OS default browser window. The frontend is not able to do that, at least in a good-looking way

Reply:
![Screenshot 2022-10-11 at 12 02 50](https://user-images.githubusercontent.com/6967377/195073717-be0f4a81-b434-48c3-bb10-be45e83dc0e4.png)

Reply:
> Since the app will display a banner at the top, I think we should also stop calling `showDialogSync`.

I was looking into the code. We are calling `showDialogSync` only when the update check was explicitly initiated by the user. On the second thought, I think we should preserve this behaviour, to let the user know their requested operation is done.

Let's close this thread as resolved.